handsontable.d.ts

```diff
@@ -2225,7 +2225,7 @@ declare namespace Handsontable {
   interface Dom {
     HTML_CHARACTERS: RegExp;
     addClass: (element: HTMLElement, className: string | any[]) => void;
-    addEvent: (element: HTMLElement, event: string, callback: () => void) => void;
+    addEvent: (element: HTMLElement, event: string, callback: (event?: Event) => void) => void;
```

Comment:
Should be `(event: Event)` not `(event?: Event)`, the `?` designates the *call* may omit the `event` argument but that's not true. The passed in callback does not need to have the event param, in other words `() => {}` is a valid callback either way.

Reply:
Thank you Aaron :) fixed